### PR TITLE
Initial play to run Django migrations.

### PR DESCRIPTION
**Pull request by doctoryes**
Adding play that will run Django migrations in a release pipeline.

@edx/pipeline-team FYI

### CODE REVIEW

**Comment by doctoryes** [comment]
@edx/pipeline-team Ready for review.

**Comment by doctoryes** [comment]
@edx/pipeline-team Please review - this is the Ansible script to run migrations.

File: playbooks/continuous_delivery/run_migrations.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      - "{{ migration_output }}"
+
+  - name: Send Hipchat notification cleanup has finished
+    hipchat_2_0_0_1:
+      api: "{{ HIPCHAT_URL }}"
+      token: "{{ HIPCHAT_TOKEN }}"
+      room: "{{ HIPCHAT_ROOM }}"
+      msg: "Migrations have completed."
+    ignore_errors: yes
+    when: hipchat_token is defined
```
Comment (feanil) [inline playbooks/continuous_delivery/run_migrations.yml]: This should be `HIPCHAT_TOKEN is defined`.  I forgot to update it when I capatilazed the rest.

**Comment by feanil** [comment]
One small change otherwise :+1: 

Reminder that this will work fine with IDAs but we will need to do some work to this to make it work for the LMS/CMS.  I think that can be looked at as separate work.

**Pull request merged**